openfisca_us/tests/baseline/finance/benefit/school_meals.yaml

```
@@ -0,0 +1,1826 @@
+        state: CA
+      child1:
+        age: 12
+      child2:
+        age: 9
+      child3:
+        age: 7
+      child4:
+        age: 5
+    spm_units:
+      spm_unit:
+        members: [parent,child1,child2,child3,child4]
+  output:
+    school_meal_subsidy: 180 * (3.26 + 1.67) * 4
+
+
+- name: Low income, one parent, four children, Hawaii
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      parent:
+        age: 40
+        gross_income: 45000
+        state: HI
+      child1:
+        age: 15
+      child2:
+        age: 9
+      child3:
+        age: 7
+      child4:
+        age: 5
+    spm_units:
+      spm_unit:
+        members: [parent,child1,child2,child3,child4]
+  output:
+    school_meal_subsidy: 180 * (4.28 + 2.29) * 4
+
+
+- name: High income, one parent, four children, Alaska
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      parent:
+        age: 40
+        gross_income: 100000
+        state: AK
+      child1:
+        age: 10
+      child2:
+        age: 9
+      child3:
+        age: 7
+      child4:
+        age: 5
+    spm_units:
+      spm_unit:
+        members: [parent,child1,child2,child3,child4]
+  output:
+    school_meal_subsidy: 180 * (0.57 + 0.50) * 4
+
+
+# 2 parents, 5 children
+
+- name: Low income, two parents, five children, contiguous_US
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      parent1:
+        age: 40
+        gross_income: 50000
+        state: CA
+      parent2:
+        age: 40
+        state: CA
+      child1:
+        age: 12
+      child2:
+        age: 10
+      child3:
+        age: 8
+      child4:
+        age: 7
+      child5:
+        age: 5
+    spm_units:
+      spm_unit:
+        members: [parent1,parent2,child1,child2,child3,child4,child5]
+  output:
+    school_meal_subsidy: 180 * (3.66 + 1.97) * 5
+
+
+- name: Income between 1.3 and 1.85 poverty line, one parent, four children, contiguous_US
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      parent1:
+        age: 40
+        gross_income: 35000
+        state: NY
+      parent2:
+        age: 40
+        gross_income: 35000
+        state: NY
+      child1:
+        age: 13
+      child2:
+        age: 11
+      child3:
+        age: 9
+      child4:
+        age: 7
+      child5:
+        age: 5
+    spm_units:
+      spm_unit:
+        members: [parent1,parent2,child1,child2,child3,child4,child5]
+  output:
+    school_meal_subsidy: 180 * (3.26 + 1.67) * 5
+
+
+- name: Income between 1.3 and 1.85 poverty line, two parents, five children, Hawaii
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      parent1:
+        age: 40
+        gross_income: 40000
+        state: HI
+      parent2:
+        age: 40
+        gross_income: 40000
+        state: HI
+      child1:
+        age: 15
+      child2:
+        age: 12
+      child3:
+        age: 10
+      child4:
+        age: 7
+      child4:
```

```suggestion
      child5:
```
this should fix the breaking test